lensless/rfft_convolve.py

```diff
@@ -68,34 +68,46 @@ def __init__(self, psf, dtype=None, pad=True, norm="ortho", **kwargs):
                 self._pad(self._psf), norm=norm, dim=(-3, -2), s=self._padded_shape[-3:-1]
             )
             self._Hadj = torch.conj(self._H)
-            self._padded_data = torch.zeros(size=self._padded_shape, dtype=dtype, device=psf.device)
```

Comment:
Maybe leave `self._padded_data=None` and say that is has to be computed/allocated on the fly for gradient tracking?